session/session.go

```diff
@@ -58,6 +58,22 @@ var (
 	SessionCount int64
 )
 
+// HandshakeClientData represents information about the client sent on the handshake.
+type HandshakeClientData struct {
+	Platform    string `json:"platform"`
+	LibVersion  string `json:"lib_version"`
```

Comment:
I think the json format should be libVersion (camel case). But if the client also is lib_version, that's not a big problem.

Reply:
It can be camel case as well. I can change it.